src/controllers/AuthController.js

```diff
@@ -82,6 +83,9 @@ class AuthController {
       keywords: `${name.givenName} ${name.familyName}`,
     });
 
+    user.user.followers = await Follow.find({ followed: user._id });
```

Comment:
Instead of doing this, it should be added under user's model everything you have been followed or unfollowed. The same thing applies for my follower